docs/reference/commandline/run.md

```diff
@@ -243,9 +247,9 @@ An example of a file passed with `--env-file`
     helloWorld=true
     TEST_PASSTHROUGH=
     HOME=/root
+    123qwe=bar
+    org.spring.config=something
 
-> **Note**: Environment variables names must consist solely of letters, numbers,
```

Comment:
I'm wondering if I remove that line *only" or if I replace it to something like :


> Environment variable names used by the utilities in the Shell and Utilities volume of IEEE Std 1003.1-2001 consist solely of uppercase letters, digits, and the '_' (underscore) from the characters defined in Portable Character Set and do not begin with a digit. **Other characters may be permitted by an implementation; applications shall tolerate the presence of such names**. Uppercase and lowercase letters shall retain their unique identities and shall not be folded together. The name space of environment variable names containing lowercase letters is reserved for applications.